core/src/main/java/org/restcomm/media/core/configuration/MgcpControllerConfiguration.java

```diff
@@ -67,6 +70,17 @@ public void setPort(int port) {
         }
         this.port = port;
     }
+
+    public int getMgcpBufferSize() {
+        return mgcpBufferSize;
+    }
+
+    public void setMgcpBufferSize(int mgcpBufferSize) {
+        if (mgcpBufferSize < 2000 || mgcpBufferSize > 10000) {
+            throw new IllegalArgumentException("Illegal MGCP buffer size value: 2000 < " + port + " < 10000");
```

Comment:
very minor one here, but you probably want to try to reformat this. Any time the exception is thrown you'll get some odd looking output (the port value will never be in the range).

Reply:
Oh sorry for typo, fixed in new commit.